app/src/main/res/layout/item_profile.xml

```diff
@@ -7,41 +7,75 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="?attr/selectableItemBackground"
-    app:cardUseCompatPadding="true"
-    app:cardPreventCornerOverlap="false"
     app:cardCornerRadius="8dp"
-    app:cardBackgroundColor="@color/purple_200">
+    app:cardPreventCornerOverlap="false"
+    app:cardUseCompatPadding="true">
 
     <ImageView
         android:id="@+id/item_image"
-        android:contentDescription="SoundWaves"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:scaleType="centerCrop"/>
+        android:background="@color/blindly_blue"
+        android:contentDescription="SoundWaves"
```

Comment:
We can add this to string resources instead of hardcoded string.

Reply:
You're right !